runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -71,7 +72,7 @@ def __init__(
         fn: Optional[Callable] = None,
```

Comment:
the class should only accept `fn_pointers`, can remove `fn`. By saving `fn_pointers` in Den that's enough to reload / share the function

Reply:
makes sense, moved the fn parsing logic from constructor to factory method.